keyboards/protokeeb/rev1/keyboard.json

```diff
@@ -0,0 +1,507 @@
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_F",
+                    "matrix": [1, 1],
+                    "x": 1,
+                    "y": 1,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_Fs",
+                    "matrix": [1, 2],
+                    "x": 2,
+                    "y": 1,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_G",
+                    "matrix": [1, 3],
+                    "x": 3,
+                    "y": 1,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_Gs",
+                    "matrix": [2, 0],
+                    "x": 0,
+                    "y": 2,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_A",
+                    "matrix": [2, 1],
+                    "x": 1,
+                    "y": 2,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_As",
+                    "matrix": [2, 2],
+                    "x": 2,
+                    "y": 2,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_B",
+                    "matrix": [2, 3],
+                    "x": 3,
+                    "y": 2,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MO(3)",
+                    "matrix": [3, 0],
+                    "x": 0,
+                    "y": 3,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "TG(1)",
+                    "matrix": [3, 1],
+                    "x": 1,
+                    "y": 3,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "TG(2)",
+                    "matrix": [3, 2],
+                    "x": 2,
+                    "y": 3,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_TOGG",
+                    "matrix": [3, 3],
+                    "x": 3,
+                    "y": 3,
+                    "w": 1,
+                    "h": 1
+                }
+            ]
+        }
+    },
+    "audio": {
+        "default": {
+            "on": false,
+            "clicky": false
+        },
+        "macro_beep": false,
+        "pins": ["GP22"],
+        "voices": false
+    },
+    "bootmagic": {
+        "enabled": true,
+        "matrix": [0, 0]
+    },
+    "dip_switch": {
+        "enabled": true,
```

Comment:
Enable at the keyboard level but disabled at all the keymap levels?

Reply:
I am using the dip switch functionality to use the encoder's built-in push button. Since the push button is not connected to the switch matrix instead it is directly connected to the GPIO pin of the RP2040 MCU. So, to use it on this keyboard I have to use the dip switch functionality to assign a fixed keycode to the push button. The user can use the push button to send a fixed keycode (KC_MUTE in this case) no matter what keymap they use with the keyboard.